### ISSUE TITLE
Provide AWS Account ID, region, partition, system tags, next token in the request to handler

### ISSUE THREAD

**Issue by wbingli** [aws-cloudformation/cloudformation-cli-go-plugin#147]
In Java plugin, there are more information exposed for the request to handler, see https://github.com/aws-cloudformation/cloudformation-cli-java-plugin/blob/master/src/main/java/software/amazon/cloudformation/proxy/ResourceHandlerRequest.java

We need include those information into the request.

**Comment by zanven42** [comment]
i currently pull out the accountid with a helper function
```func getAccountID(sess *session.Session) (AccountID string, err error) {
	stsSess := sts.New(sess)
	identity, err := stsSess.GetCallerIdentity(&sts.GetCallerIdentityInput{})
	if err != nil {
		return "", err
	}
	if identity.Account != nil {
		AccountID = *identity.Account
		return
	}
	return "", errors.New("AccountID invalid for session")
}
```
and just pass in the `req.session`

i would really like to get the StackID for the execution so i can create tags similar to AWS resources with the source stack arn, and / or append stack names in resource names created.

**Comment by zanven42** [comment]
https://github.com/aws-cloudformation/cloudformation-cli-go-plugin/blob/c159be97639a5393e9bc3cc683ddd8b125f85ae1/cfn/event.go#L23

would be awesome if this ticket also handled providing the stackId

**Comment by brianterry** [comment]
Fixed in PR #157
